frontend/src/hooks/useFormatTableData.tsx

```diff
@@ -97,10 +97,7 @@ export const useFormatTableData = ({
 
   // Queries
   //Destructure the keycloak functionality
-  /* eslint "no-warning-comments": [1, { "terms": ["todo", "fixme"] }] */
-  // todo: Define a good type. "Any" type temporarily permitted.
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  const { data, isLoading } = useQuery<any>([apiEndPoint], getTableData, {
+  const { data, isLoading } = useQuery([apiEndPoint], getTableData, {
```

Comment:
does this not need a type?

Reply:
nope. the linter does not complain, so  its safe to conclude the typescript engine can infer the type automatically

Reply:
It turns out that the library already has a type defined by the author, so typescript infers it for us. I'm glad you asked though, I'd forgotten about that detail in Typescript